### ISSUE TITLE
allow user to connect to VM screen

### ISSUE THREAD

**Issue by K0zka** [kerubistan/kerub#77]
spice integration

**Comment by K0zka** [comment]
https://people.freedesktop.org/~teuf/spice-doc/html/ch03.html

**Comment by K0zka** [comment]
blocked by #142

**Comment by K0zka** [comment]
let's call this good enough for now, improvements will come under another ticket
